lang/RStudio/RStudio.def

```diff
@@ -1,21 +1,22 @@
 Bootstrap: library
-From: debian:9
+From: debian:10
 
 %environment
     LC_ALL="C"
     export LC_ALL
 
 %post
     apt-get update
-    apt-get -y install systemd systemd-sysv gdebi-core procps libssl1.1 ed wget curl libqt5webkit5 libqt5core5a
-    apt-get -y install r-base r-base-dev
+    apt -y install systemd systemd-sysv gdebi-core procps libssl1.1 libnss3 \
+           libasound2 ed wget curl libqt5webkit5 libqt5core5a r-base r-base-dev
 
-    curl https://download1.rstudio.org/rstudio-xenial-1.1.456-amd64.deb > /rstudio-1.1.456-amd64.deb
-    apt-get -y install /rstudio-1.1.456-amd64.deb
+    wget -O /rstudio-2023.03.0-386-amd64.deb \
+      https://download1.rstudio.org/electron/bionic/amd64/rstudio-2023.03.0-386-amd64.deb
+    apt -y install /rstudio-2023.03.0-386-amd64.deb
 
-    wget -O /rstudio-server-stretch-1.1.456-amd64.deb \
-      https://download2.rstudio.org/rstudio-server-stretch-1.1.456-amd64.deb
-    gdebi -n /rstudio-server-stretch-1.1.456-amd64.deb
+    wget -O /rstudio-server-2023.03.0-386-amd64.deb \
+      https://download2.rstudio.org/server/bionic/amd64/rstudio-server-2023.03.0-386-amd64.deb 
```

Comment:
It's probably best to update to Ubuntu 22.04 as the image source, and use the 'jammy' RStudio packages if you can.

Ubuntu bionic goes EOL soon... so it's likely that RStudio won't have newer packages in the near future.

Also, Debian 10 is now oldstable... and drops out of LTS next year... https://wiki.debian.org/LTS